### ISSUE TITLE
BUG - The plugin makes IOS app not working

### ISSUE THREAD

**Issue by salsudais** [standup-raven/standup-raven#142]
**Summary**
When we enable the plugin, the IOS gives a white page (attached a screenshot) and after we disable the plugin, the app works fine.

**Steps To Reproduce**
1. Enable the plugin.
2. Open the IOS app.
3. The app will give you a white page.

**Expected behavior**
The IOS shouldn't be affected.

**Observed Behavior**
IOS shows a white page.

**Screenshots**
![image](https://user-images.githubusercontent.com/62443977/77357212-d5d0c500-6d58-11ea-8f85-26ba6af7bea6.png)

**Environment**
-  Mattermost version: 5.18.0
-  OS: iOS
-  Platform: mobile app
-  Browser (if applicable): 
-  Platform Version: IOS 13.3.1

**Comment by harshilsharma63** [comment]
@salsudais thanks for the bug report.

I'll try reproducing and debug the bug and it might take some time since I don't have access to an iPhone. You can access it on the mobile browser meanwhile if it helps.

**Comment by chetanyakan** [comment]
I tested this on `iPhone 7+` and `iOS 13.3.1` with the latest version of mattermost mobile app from the app store.
- I was able to connect to our organization's mattermost server, which has `standup-raven` enabled and configured, without any issues.
- The functionality of the plugin, however, was limited as the webapp components aren't loaded in the mobile apps.

I also tested with safari browser on iOS and the app worked there as well.
